pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[build-system]
+requires = ["scikit-build-core", "swig"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "gappa"
+version = "1.0.0"
```

Comment:
@MischaBr Before merging or at least before publishing, we should add more project metadata here